scripts/cli/handler/parse.py

```diff
@@ -0,0 +1,377 @@
+import datetime
+import glob
+import os
+import re
+import shutil
+import subprocess
+import sys
+import time
+import signal
+
+from os.path import join
+
+from util.configfilehandler import ConfigFileHandler
+from util import util
+
+def __current_time():
+  '''This function returns the current date and time as string.'''
+  return datetime.datetime.fromtimestamp(
+    time.time()).strftime('%Y-%m-%d %H:%M:%S')
+
+def __handle_parser_crash(projectdir):
+  #--- Get line containing crash message in the log file ---#
+
+  with open(join(projectdir, 'parselog'), 'r') as f:
+    for line in f:
+      if 'CODECOMPASS CRASH' in line:
```

Comment:
We had a `crashprotect.cpp` file in the old CC where we wrote these messages. We don't support it now.